.github/workflows/ci.yml

```
@@ -138,8 +139,15 @@ jobs:
           # Use smart diff coverage analyzer
           chmod +x scripts/analyze-diff-coverage.sh
           ./scripts/analyze-diff-coverage.sh origin/main build/reports/kover/report.xml
+        continue-on-error: true
+      - uses: mainmatter/continue-on-error-comment@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          outcome: ${{ steps.diff_coverage.outcome }}
+          test-id: This PR does not satisfy 80% diff coverage ${{ matrix.code }}
```

actionlint (1.7.10)

[error] property "code" is not defined in object type {os: string}

(expression)